src/main/controllers/NoticeEndController.ts

```diff
@@ -11,7 +11,35 @@ import { assignFormData, getPageContent, handleSessionErrors, setUserCase } from
 export default class NoticeEndController {
   private readonly form: Form;
   private readonly noticeEndContent: FormContent = {
-    fields: {},
+    fields: {
+      noticeEnd: {
+        id: 'notice-end',
+        type: 'date',
+        classes: 'govuk-date-input',
+        label: (l: AnyRecord): string => l.label,
+        labelHidden: true,
```

Comment:
If you look under the definitions folder and the date.ts file, you should see some code there which you can resuse/extend for date fields.